core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -120,11 +120,48 @@ var SkillEditorPage = function() {
     });
   };
 
-  this.expectRubricExplanationToMatch = function(index, explanation) {
-    rubricExplanations.then(function(explanations) {
-      explanations[index].getText().then(function(text) {
-        expect(text).toMatch(explanation);
-      });
+  this.deleteRubricExplanationWithIndex = function(diffIndex, explIndex) {
+    var editRubricExplanationButtons = element.all(
+      by.css('.protractor-test-edit-rubric-explanation-' + diffIndex));
+    editRubricExplanationButtons.then(function(buttons) {
```

Comment:
At the top of the diff for this file, it looks like you deleted `editRubricExplanationButtons`. If this doesn't exist, how did this pass?

Reply:
I have declared it above this line. I removed it from there, since the class name depends on the difficulty index (will change to difficulty name).